spanner/spanner_snippets/spanner/spanner_update_database.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package spanner
+
+// [START spanner_update_database]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	database "cloud.google.com/go/spanner/admin/database/apiv1"
+	adminpb "cloud.google.com/go/spanner/admin/database/apiv1/databasepb"
+	"google.golang.org/genproto/protobuf/field_mask"
+)
+
+func updateDatabase(ctx context.Context, w io.Writer, db string) error {
+	// db = `projects/<project>/instances/<instance-id>/database/<database-id>`
+	// Instantiate database admin client.
+	adminClient, err := database.NewDatabaseAdminClient(ctx)
+	if err != nil {
+		return fmt.Errorf("updateDatabase.NewDatabaseAdminClient: %v", err)
```

Comment:
nit: I'd reference the function including the import name rather than the caller function; e.g. `database.NewDatabaseAdminClient`. Same on L48 & L54 below

Reply:
That is a good suggestion @tritone.
But, I feel printing `updateDatabase.NewDatabaseAdminClient` will help us quickly understand which sample is throwing error when the tests run. 
`database.NewDatabaseAdminClient` could be used in multiple samples and we might need to go through the entire stack trace to understand which sample is the error occurring in.

Also this has been followed for the few existing samples https://github.com/GoogleCloudPlatform/golang-samples/blob/main/spanner/spanner_snippets/spanner/spanner_create_database_with_default_leader.go#L70